src/main/java/org/fundation/coding/adrian/katas/cyphers/CesarCrypto.java

```diff
@@ -0,0 +1,45 @@
+package org.fundation.coding.adrian.katas.cyphers;
+
+import java.util.ArrayList;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.ArrayList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)